app/kuma-dp/pkg/dataplane/metrics/server.go

```diff
@@ -33,6 +33,11 @@ import (
 var (
 	inPassThroughIPv4 = &net.TCPAddr{IP: net.ParseIP("127.0.0.6")}
 	inPassThroughIPv6 = &net.TCPAddr{IP: net.ParseIP("::6")}
+	fmtOpenMetrics_1_0_0 = expfmt.NewFormat(expfmt.TypeOpenMetrics)
+	// for some reason 0.0.1 is not taken into account in expfmt.NewFormat
```

Comment:
https://github.com/prometheus/common/blob/v0.48.0/expfmt/expfmt.go#L87 and https://github.com/prometheus/common/blob/v0.47.0/expfmt/expfmt.go#L50